pkg/kubectl/cmd/util/openapi/openapi_cache_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package openapi
+package openapi_test
```

Comment:
Yes, to match the name of the suite.  I forget the tradeoffs, but this is the recommended way for ginkgo